### DISCUSSION TITLE
Meshtastic

### DISCUSSION THREAD

**Discussion by barefootpookie8** [selfhst/icons#123] [New Icon Request]
GitHub: https://github.com/meshtastic
Official: https://meshtastic.org/
SVG: https://github.com/meshtastic/design/blob/77fafb2b027787f1206fe3774a2f456f1ffc721d/logo/logo.svg
PNG: https://github.com/meshtastic/design/blob/77fafb2b027787f1206fe3774a2f456f1ffc721d/logo/png/play_store_icon.png
Logo Landing Page (Github): https://github.com/meshtastic/design/tree/77fafb2b027787f1206fe3774a2f456f1ffc721d/logo

**Comment by shollyethan** [comment]
Meshtastic has been added - thanks!
